lib/os/fdtable.c

```diff
@@ -189,7 +230,19 @@ int close(int fd)
 
 	return res;
 }
-FUNC_ALIAS(close, _close, int);
+
+#ifdef CONFIG_USERSPACE
+ssize_t z_vrfy__close(int fd)
+{
+	return z__impl_close(fd);
```

Comment:
`z__impl_close(fd)` this has to be a typo, the function is called `z_impl__close`.

Does this code have tests which run these system calls from user mode?

Reply:
Thanks, typo fixed. Regarding tests, see comment above (enabling them wip).